packages/phpstan-rules/src/Rules/ForbiddenFuncCallRule.php

```diff
@@ -89,9 +89,86 @@ class SomeClass
                     'forbiddenFunctions' => ['eval'],
                 ]
             ),
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+class SomeClass
+{
+    dump('hello world');
+    return true;
+}
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+class SomeClass
+{
+    return true;
+}
+CODE_SAMPLE
+            ,
+                [
+                    'forbiddenFunctions' => ['dump' => 'seems you missed some debugging function'],
+                ]
+            ),
         ]);
     }
 
+    private function formatError(string $funcName): string {
+        foreach($this->getForbiddenFunctionsWithMessages() as $forbiddenFunction => $additionalMessage) {
+            if (!$additionalMessage) {
+                continue;
+            }
+
+            if (! $this->arrayStringAndFnMatcher->isMatch($funcName, [$forbiddenFunction])) {
+                continue;
+            }
+
+            return sprintf(self::ERROR_MESSAGE .': '. $additionalMessage, $funcName);
+        }
+        return sprintf(self::ERROR_MESSAGE, $funcName);
+    }
```

Comment:
Could you extract this to a standalone service? We'll have a few places with optional comments to re-use it

Reply:
sure thing. where should such a class be placed into? will this class only contain the 3 lines 
```
            return sprintf(self::ERROR_MESSAGE .': '. $additionalMessage, $funcName);
        }
        return sprintf(self::ERROR_MESSAGE, $funcName);
```

wrapped in a method? asking, because the `! $this->arrayStringAndFnMatcher->isMatch($funcName, [$forbiddenFunction])` part seems not to be re-usable after all?

Reply:
This whole method should be extracted, with all its deps.

Try to check other classes and location and pick your best shot.